static/bidder-info/criteo.yaml

```diff
@@ -14,9 +14,9 @@ capabilities:
 userSync:
   # criteo supports user syncing, but requires configuration by the host. contact this
   # bidder directly at the email address in this file to ask about enabling user sync.
-  supports:
-    - redirect
-
   redirect:
     url: "https://ssp-sync.criteo.com/user-sync/redirect?gdprapplies={{.GDPR}}&gdpr={{.GDPRConsent}}&ccpa={{.USPrivacy}}&redir={{.RedirectURL}}&profile=230"
+    userMacro: "${CRITEO_USER_ID}"
+  iframe:
+    url: "https://ssp-sync.criteo.com/user-sync/iframe?gdprapplies={{.GDPR}}&gdpr={{.GDPRConsent}}&ccpa={{.USPrivacy}}&redir={{.RedirectURL}}&profile=230"
```

Comment:
Verified. Both redirects and iframe urls 
are working correctly
```
Redirect
https://ssp-sync.criteo.com/user-sync/redirect?gdprapplies=&gdpr=&ccpa=&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dcriteo%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24%7BCRITEO_USER_ID%7D&profile=230
&profile=230

Iframe:
https://ssp-sync.criteo.com/user-sync/iframe?gdprapplies=&gdpr=&ccpa=&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dcriteo%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24%7BCRITEO_USER_ID%7D
&profile=230

```
<img width="254" height="128" alt="Screenshot 2023-05-17 at 12 04 45 PM" src="https://github.com/prebid/prebid-server/assets/24757781/7a4597de-4f69-4720-a8ca-928bb313fb7c"><img width="254" height="129" alt="Screenshot 2023-05-17 at 12 06 41 PM" src="https://github.com/prebid/prebid-server/assets/24757781/8c3c34b5-43a6-4df1-b605-d2a42db704b6">